pupil-spa/src/app/question/question.component.ts

```diff
@@ -87,7 +87,7 @@ export class QuestionComponent implements OnInit, AfterViewInit {
    */
   @HostListener('document:keydown', ['$event'])
   handleKeyboardEvent(event: KeyboardEvent) {
-    console.log('question.component: handleKeyboardEvent(): event: ', event);
+    // console.log('question.component: handleKeyboardEvent(): event: ', event);
```

Comment:
will you need this console log?
can always revert from git history.
I'd rather keep it as clean and 'comment-less' as possible

Reply:
Yes - these will be replaced by a call to a logger Real Soon Now